config/kernel/linux-rockchip64-edge.config

```diff
@@ -2334,6 +2331,9 @@ CONFIG_SND_AUDIO_GRAPH_CARD2_CUSTOM_SAMPLE=m
 CONFIG_SND_TEST_COMPONENT=m
 CONFIG_SND_XEN_FRONTEND=m
 CONFIG_SND_VIRTIO=m
+CONFIG_HID_SUPPORT=y
+CONFIG_HID=y
+CONFIG_HID_GENERIC=m
```

Comment:
USB HID devices may not work on photonicat2 after boot, adding these configs seems to made it stable.